.github/ISSUE_TEMPLATE/feature_request.md

```
@@ -1,20 +1,49 @@
+### Must NOT Do
+- Don't change existing behavior
+- Don't add extra dependencies without approval
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Out of Scope
+- <!-- Related but separate work — future issue -->
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+## Acceptance Criteria
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)